lifecycle_msgs/msg/State.msg

```diff
@@ -1,26 +1,57 @@
-# Primary state definitions
-# as depicted in
+# Primary state definitions as depicted in
```

Comment:
```suggestion
# Primary state definitions as depicted in.
```

Reply:
Addressed in https://github.com/ros2/rcl_interfaces/pull/91